vue.config.js

```diff
@@ -4,7 +4,17 @@ module.exports = {
   lintOnSave: false,
   runtimeCompiler: true,
   css: {
-    extract: false
+    extract: false,
+    loaderOptions: {
+      sass: {
+        prependData: `
+            @import './node_modules/bootstrap/scss/_functions.scss';
+            @import './node_modules/bootstrap/scss/_variables.scss';
+            @import './node_modules/bootstrap/scss/_mixins.scss';
+            @import '@/scss/bootstrap3_compatibility.scss';
```

Comment:
@safydy `prependData` feels a bit like a last-resort type of thing to me. Can you help me understand what the benefits of dropping this here as opposed to requiring a SASS file from `src/main.ts` that carries this same business?

Reply:
hum, it's about maintainability I would say.
Putting scss files in `prependData` allows us to not worry about storybook's scss dependencies. I feel it's very easy to miss updating the stories entry points with these imports (such as `stories/story-utils/FlowBuilderContainer.vue`, `stories/story-utils/PlainFlowBuilderBlockEditorContainer.vue`). In fact, in the future we may add new storybook entry points OR add another scss dependencies. (I only thought about scss at that time, but we can think about css files as well)

Is there any drawback if we choose `prependData` ?

Reply:
After quick catchup: Will create one single file to import all of them, and won't use the prependData